modes/longitudinal/src/index.js

```diff
@@ -13,25 +14,31 @@ const tracked = {
   viewport: 'org.ohif.measurement-tracking.viewportModule.cornerstone-tracked',
 };
 
-const dicomsr = {
-  sopClassHandler: 'org.ohif.dicom-sr.sopClassHandlerModule.dicom-sr',
-  viewport: 'org.ohif.dicom-sr.viewportModule.dicom-sr',
-};
 
-const dicomvideo = {
-  sopClassHandler: 'org.ohif.dicom-video.sopClassHandlerModule.dicom-video',
-  viewport: 'org.ohif.dicom-video.viewportModule.dicom-video',
-}
+const { longitudinalMode } = ConfigPoint.register({
+  longitudinalMode: {
+    configBase: "modeConfig",
+    // Add custom settings for tracked measurements
+    panels: {
+      leftPanels: [tracked.thumbnailList],
+      rightPanels: [tracked.measurements],
+      viewports: {
+        images: {
+          namespace: tracked.viewport,
+          displaySetsToDisplay: [ohif.sopClassHandler],
+        },
+      },
+      // Can customize sopClassHandlers here if desired
+    },
+  },
+});
 
-export default function mode({ modeConfiguration }) {
-  return {
-    // TODO: We're using this as a route segment
-    // We should not be.
+const { longitudinalMode: updatedLongitudinalMode } = ConfigPoint.register({
```

Comment:
Doesn't appear this is used yet, but what was the intention? Can we remove this if its not used or is there more to come in this PR?

Reply:
Added a comment inline about why a second register occurs, and removed the actual second variable, which isn't needed.

Reply:
> Another thought is that a limitation of this approach is that mode schema is leaking into extension schema.
The mode schema is already coupled to how it consumes the extensions, as they are explicitly listed in the mode itself with the viewport being linked by name to the sop class handler, and the list of sop class handlers being listed - I would argue that config point actually reduces the coupling because the extension only needs to understand the part of the mode schema that it is enhancing.  The extension does specify that it modifies the mode by adding the schema and the viewport, but that is all it specifies, for example, the pdf extension only lists the viewport and sop class handlers sections:
modeConfig: {
    panels: {
      viewports: {
        pdf: {
          id,
          namespace: ViewportModuleId,
          displaySetsToDisplay: [SOPClassHandlerId],
        },
      },
    },
    sopClassHandlers: {
      pdf: { id: SOPClassHandlerId, priority: 5000 },
    },

Now, suppose we have another mode that doesn't want any default handling and uses a completely new schema for extensions.  The new mode is just defined the old way, and explicitly lists all the extensions it wants included in whatever schema format it wants.  Yes, it is now tightly coupled to the extensions it uses, but it still works just fine.  
Or alternatively, suppose a new global schema is defined for new modes - it can use config point to define the new schema, and extensions that get updated can plugin to the new schema, OR the new schema can specifically list the new default extensions it wants to include (or any combination of the two).  The extensions supporting the new schema can now be used with both the old and new schemas by just listing both configurations.  Config point is designed to support shared configuration, but it isn't required anywhere, just that it makes the changes a bit more standard and more decoupled when it does get used.  If one were clever, it would be possible to write extensions that work on both OHIF v2 and OHIF v3 by registering different functions into config-point.

Reply:
> The new mode is just defined the old way, and explicitly lists all the extensions it wants included in whatever schema format it wants. 

Thanks for this point especially, I think this is something I didn't realise. I'll mull over your thoughts.